libnetwork/drivers/bridge/setup_ipv6_linux.go

```diff
@@ -6,15 +6,11 @@ import (
 	"os"
 )
 
+const disableIPv6ConfPerm = 0o644
```

Comment:
Perhaps use the same const as for IPv4 (`forwardConfPerm`) or even consider removing the const entirely.

Reply:
Removed.